networkit/cpp/generators/MocnikGenerator.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * MocnikGenerator.h
+ *
+ *  Created on: July 7, 2018
+ *      Author: Franz-Benjamin Mocnik <mail@mocnik-science.net>
+ */
+
+#ifndef MOCNIKGENERATOR_H_
+#define MOCNIKGENERATOR_H_
+
+#include "StaticGraphGenerator.h"
+#include <unordered_map>
+
+namespace NetworKit {
+
+/**
+ * @ingroup generators
+ * Creates G(d, n, p) graphs.
+ */
+class MocnikGenerator: public StaticGraphGenerator {
+private:
+    typedef std::vector<std::pair<node, std::vector<double>>> NodePositionMap;
```

Comment:
What does this actually store? Please add a comment what the first/second component stores?

Reply:
comment added